### PR TITLE
PEP 8 and more comments on `example.py`

### PR DESCRIPTION
**Pull request by maxbellec**

### CODE REVIEW

**Comment by maxbellec** [comment]
It seems I've broken the Github by accidentally pushing to master than reverting. This is the actual diff : https://github.com/tmrowco/electricitymap/compare/aaff26996fd2113d8c2952224873f504a2a18c67...68643c3

**Comment by corradio** [comment]
Can you merge latest master onto this branch?

File: parsers/example.py [hunk 1]
```diff
@@ -175,16 +219,17 @@ def fetch_exchange(zone_key1='DK', zone_key2='NO', session=None, target_datetime
         'source': 'someservice.com',
     }
 
-    # Country codes are sorted in order to enable easier indexing in the database
+    # Country codes are sorted in order to enable easier indexing in the DB
```
Comment (corradio) [inline parsers/example.py]: very minor comment: I think acronyms should be avoided here.

File: parsers/example.py [hunk 2]
```diff
@@ -73,35 +91,48 @@ def fetch_production(zone_key='FR', session=None, target_datetime: datetime.date
...
 
     # Parse the datetime and return a python datetime object
     data['datetime'] = arrow.get(obj['datetime']).datetime
 
     return data
 
 
-def fetch_price(zone_key='FR', session=None, target_datetime=None, logger=None):
+def fetch_price(zone_key='FR', session=None, target_datetime=None,
+                logger=None):
```
Comment (corradio) [inline parsers/example.py]: Maybe in example.py we could set a default logger here?

**Comment by corradio** [comment]
Few (very) minor comments. Feel free to merge/remove branch when you feel it!

**Pull request merged**